make check has errors

specifically 
```
$ make check
python tools/check.py .
WARNING: In ./04-loop.md: Sub-headings are often a sign a lesson needs to be split into multiple topics. Please make sure this subsection doesn't belong in a separate lesson.
WARNING: In ./05-script.md: Sub-headings are often a sign a lesson needs to be split into multiple topics. Please make sure this subsection doesn't belong in a separate lesson.
ERROR: In ./01-filedir.md: Heading at line 364 should be level 2
WARNING: In ./01-filedir.md: Sub-headings are often a sign a lesson needs to be split into multiple topics. Please make sure this subsection doesn't belong in a separate lesson.
WARNING: In ./00-intro.md: Sub-headings are often a sign a lesson needs to be split into multiple topics. Please make sure this subsection doesn't belong in a separate lesson.
ERROR: In ./03-pipefilter.md: Heading at line 290 should be level 2
WARNING: In ./03-pipefilter.md: Sub-headings are often a sign a lesson needs to be split into multiple topics. Please make sure this subsection doesn't belong in a separate lesson.
WARNING: 2 errors were encountered during validation. See log for details.
make: *** [check] Error 2
```

--- a/01-filedir.md
+++ b/01-filedir.md
@@ -361,7 +361,7 @@ Finally, we also see a file called `.bash_profile`. This file usually contains s
 > Orthogonal systems tend to be easier for people to learn
 > because there are fewer special cases and exceptions to keep track of.
 
-### Nelle's Pipeline: Organizing Files
+## Nelle's Pipeline: Organizing Files
 
 Knowing just this much about files and directories,
 Nelle is ready to organize the files that the protein assay machine will create.

--- a/03-pipefilter.md
+++ b/03-pipefilter.md
@@ -287,7 +287,7 @@ so that you and other people can put those programs into pipes to multiply their
 > have told the shell to send the contents of `ammonia.pdb` to `wc`'s
 > standard input.
 
-### Nelle's Pipeline: Checking Files
+## Nelle's Pipeline: Checking Files
 
 Nelle has run her samples through the assay machines
 and created 1520 files in the `north-pacific-gyre/2012-07-03` directory described earlier.

